freezeyt/cli.py

```diff
@@ -34,10 +34,12 @@
 @click.option('--gh-pages/--no-gh-pages', 'gh_pages',
               default=None,
               help='If activated and freeze was successful, create git gh-pages branch in output folder and commit all files to that branch.')
-
+@click.option('-x', '--fail-fast/--no-fail-fast',
```

Comment:
Maybe we can add just as flag, but if we want to use also in configuration file this approach seems made more sense

we keep state if fail fast is set or not 
we can explicitly turn off from CLI, it could be more user friendly for debug if someone forget in config (I am not sure)
We keep consistency in code.
Implementation of handling that option is easy.

Reply:
For boolean flags, it's best to just always add the `--no-` variant. Unless there's some good reason against it, of course.